app/search_builders/curation_concerns/filter_suppressed_with_roles.rb

```diff
@@ -0,0 +1,27 @@
+module CurationConcerns
+  module FilterSuppressedWithRoles
+    extend CurationConcerns::FilterSuppressed
+
+    def only_active_works(solr_parameters)
+      if current_work.respond_to?(:to_sipity_entity) # Used because Collections use WorkSearchBuilder to filter down to a single item
+        super if user_work_permissions.empty?
```

Comment:
Per @jcoyne's above comment, the `super`s in this module do not wind up calling the method with the same name in `FilterSuppressed` unless the class you're calling this method on *also* includes `FilterSuppressed`.